### ISSUE TITLE
Use runkit for most code examples in guides

### ISSUE THREAD

**Issue by edanchenkov** [PlatformOfTrust/collected-feedback#26]
# Is your suggestion for an existing guide? Or you are requesting a new guide?
I suggest to use runkit for requests even for CURL (via Node if needed). First of all they are interactive and easy to use. Secondly it allows to create better structured code examples with variables that can be easily replaced in browser by users.

**Comment by Nazarah** [comment]
@edanchenkov  good idea. should we start with the master guides? e.g. Data Product 101 and Register an Application? Or do you have specific lists in your mind?

**Comment by edanchenkov** [comment]
We can start with https://developer.oftrust.net/guides/identities-and-links/ guide which requires update anyway, and it actually it is a perfect example for runkit and code snippets with variables where users have to anyway change values in command.

**Comment by Nazarah** [comment]
@edanchenkov  I will start the task of updating the guide in next sprint. if you could work on the runkit modules, that would be great.

**Comment by edanchenkov** [comment]
I need to explore functionality and usability of runkit modules and see what can be needed for us This can take a day or so. However I cannot work on anything outside of my sprint tasks. Unless they somehow related to existing features in this PI.

**Comment by Nazarah** [comment]
@edanchenkov if this needs extra work, then I'll suggest adding it as a feature. To fix the guide and then add possible runkit modules for it. sound ok?

**Comment by edanchenkov** [comment]
Yes, it should be a separate task at least. I would like first to explore runkit and then with required knowledge update the guide.

**Comment by Nazarah** [comment]
@edanchenkov  you can think about it for the IP sprint if your time allocates.
